syz-fuzzer/fuzzer.go

```diff
@@ -417,7 +417,7 @@ func triageInput(pid int, env *ipc.Env, inp Input) {
 		}
 		minCover = cover.Intersection(minCover, cov)
 		return true
-	})
+	}, false)
```

Comment:
I don't know if this one must be true, or the one in repro since I don't understand meaning of 'full'. But one of them must be true.